### ISSUE TITLE
Unable to create machine (Shared folder)

### ISSUE THREAD

**Issue by Nox-404** [Parallels/docker-machine#7]
- parallels : ```Version 11.0.1 (31277)```
- parallels/docker-machine : ```docker-machine version 0.4.0-1 (1981be6)```

```bash
Using SSH client type: external
About to run SSH command:
sudo mkdir -p /Users && sudo mount -t prl_fs Users /Users
&{/usr/bin/ssh [/usr/bin/ssh -o PasswordAuthentication=no -o IdentitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet -o ConnectionAttempts=3 -o ConnectTimeout=10 -i /Users/aurelien/.docker/machine/machines/prl-dev/id_rsa -p 22 docker@10.211.55.6 sudo mkdir -p /Users && sudo mount -t prl_fs Users /Users] []  <nil> <nil> <nil> [] <nil> <nil> <nil> <nil> false [] [] [] [] <nil>}
SSH cmd err, output: exit status 255: mount: mounting Users on /Users failed: No such device

Error creating machine: Error mounting shared folder: exit status 255
You will want to check the provider to make sure the machine and associated resources were properly removed.
```

**Comment by legal90** [comment]
@Nox-404 Please provide more details about this issue:
* What command have you used to create this VM?
* Have you tried to restart your VM? (`docker-machine restart <vm_name>`)
* Try to login there by ssh (`docker-machine ssh <vm_name>`) - what Boot2Docker version and build number is displayed in the prompt?
* Try to create new VM from the scratch - is this error occurred again?

**Comment by Nox-404** [comment]
- I have used this to create the machine : ```docker-machine create --driver=parallels docker --debug```
- Restarting the VM didn't fix it
```bash
Waiting for VM to start...
Error mounting shared folder: exit status 255
Restarted machines may have new IP addresses. You may need to re-run the `docker-machine env` command.
```
- when I ssh, i get these versions : 
```bash
Boot2Docker version 1.8.3, build master : af8b089 - Mon Oct 12 18:56:54 UTC 2015
Docker version 1.8.3, build f4bf5c7
```
- I already tried several times

**Comment by FreshXOpenSource** [comment]
I have the same issue here. My setup:

OSX 10.10.4
Parallels 11.0.2 (31348)
docker-machine version 0.4.0-1 (1981be6)

The output of the failed command:
```
&{/usr/bin/ssh [/usr/bin/ssh -o PasswordAuthentication=no -o IdentitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet -o ConnectionAttempts=3 -o ConnectTimeout=10 -i /Users/dominik/.docker/machine/machines/prl-dev/id_rsa -p 22 docker@10.211.55.16 sudo mkdir -p /Users && sudo mount -t prl_fs Users /Users] []  <nil> <nil> <nil> [] <nil> <nil> <nil> <nil> false [] [] [] [] <nil>}
SSH cmd err, output: exit status 255: mount: mounting Users on /Users failed: No such device
```

The version of Boot2Docker:
```
Boot2Docker version 1.8.3, build master : af8b089 - Mon Oct 12 18:56:54 UTC 2015
Docker version 1.8.3, build f4bf5c7
````

I can also reproduce the error by execution the mount command directly on the machine:
```
docker@boot2docker:~$ sudo mount -t prl_fs Users /Users
mount: mounting Users on /Users failed: No such device
docker@boot2docker:~$
```

**Comment by legal90** [comment]
Yep, seems like Parallels Tools are broken in Boot2Docker v1.8.3. Filesystem `prl_fs` is unavailable. I'm investigating this issue.

P.s. For now you can use previous version, 1.8.2, it should work fine:
```
docker-machine create \
  -d parallels \
  --parallels-boot2docker-iso https://github.com/boot2docker/boot2docker/releases/download/v1.8.2/boot2docker.iso \
  <your_vm_name>
```

**Comment by FreshXOpenSource** [comment]
Thanks, it's working now. But I think you meant --parallels-boot2docker-url instead of --paralles-boot2docker-iso:
```
$ docker-machine create \
  -d parallels \
  --parallels-boot2docker-url https://github.com/boot2docker/boot2docker/releases/download/v1.8.2/boot2docker.iso \
  <your_vm_name>
```

**Comment by legal90** [comment]
@FreshXOpenSource Yes, sure. I've edited my message. Thank you!

**Comment by Nox-404** [comment]
@legal90 It works, thanks

**Comment by legal90** [comment]
I reopen it, because this issue is not fixed yet.

**Comment by mhenrixon** [comment]
:+1:

**Comment by legal90** [comment]
This issue has been fixed in Boot2Docker v1.9.0
So, all you need to do is just upgrade your machine (it will automatically download and cache the latest boot2docker ISO):
```console
$ docker-machine upgrade <vm_name>
```
